tmpmail: add page (#5026)
tldr-pages/tldr: pages/common/tmpmail.md

# tmpmail

> A temporary email right from your terminal written in POSIX sh.
> More information: <https://github.com/sdushantha/tmpmail>.

> - Create a temporary email:

Reviewer: I'm not sure if we should call this `inbox`:
```suggestion
- Create a temporary inbox:
```

`tmpmail --generate`

- List messages and their numeric ID:

`tmpmail`

> - View a temporary email's most recent email:

Reviewer: Is this for received emails right? (no sent)
```suggestion
- Display the most recent received email:
```

`tmpmail --recent`

> - Open user specified message:

Reviewer: ```suggestion
- Open a specific message:
```

`tmpmail {{email_id}}`

- View email as raw text without HTML tags:

`tmpmail --text`

> - Open email with given browser (default is w3m):

Reviewer: ```suggestion
- Open email with a specific browser (default is w3m):
```
